hummingbot/strategy/pure_market_making/pure_market_making_config_map.py

```diff
@@ -34,7 +34,12 @@ def is_valid_maker_market_symbol(value: str) -> bool:
                                                   prompt="What is your preferred quantity per order (denominated in "
                                                          "the base asset, default is 1)? >>> ",
                                                   default=1.0,
-                                                  type_str="float"),
+                                                 type_str="float"),
```

Comment:
Feels like it's gonna be a multiple-level choice here, instead of just once level. If you try to ask all questions for all plugins here, then most of the questions here will be irrelevant for users.

We're gonna need a different question sequence rather than trying to ask the user everything up-front.

(After choosing pure market making strategy)
- Do you want to configure pricing plugin?
- Do you want to configure sizing plugin?
(Assuming yes to the above questions)
- Which pricing plugin?
- Which sizing plugin?
- (plugin-specific questions)

The plugin maker will likely need to come up with the config options and include those in a specific variable name in his module. We'll need to discuss w/ @yzhang1994 on how to do this.

Reply:
For v0.9.0... we probably don't need to implement the full plugin config architecture. But we should start by asking the initial questions like "Do you want to configure staggered orders for market making?". If it's no then we proceed with the old constant spread / size questions. Basically we should start by having the multiple-level choice architecture.

Then in the next next version (say, 0.10.0)... we can build incrementally upon that and have a more complete architecture.